doc/ref_model/chapters/chapter01.md

```diff
@@ -238,7 +243,98 @@ The ratings are as follows:
+
+The following subsections, Executive Summary, Strategy Objectives, Networking Requirements and Initial Approach, respectively, are intended to provide an overview of CNTT's vision for networking and intent, objectives, requirements and their supporting rationale, as well as initial development approach.
+
+The networking within an NFVI, fabric or otherwise, is an area where there is significant variability across implementations. Leaf-Spine topology is well established, however, after topology there are countless decisions an Operator needs to make. Differences arise from many aspects, for example, is the solution layer-2 or layer-3; is the routing static or dynamic; what mechanism is used for encapsulation; what mechanism is used for isolation; does it support SR-IOV; does it support DPDK; does it employee SmartNICs; does it employ distributed control or a centralized control driving a programmable fabric; and the list continues. The multitude of permutations enable Operators to design (or procure) a fabric/networking solution that's optimized for their needs, whether their needs are minimal, very extensive or somewhere in between.
+
+For CNTT, a strategy is needed that affords Operators the performance, flexibility, availability, maintainability and scalability their business requires, yet doesn't require OPNFV to design, manage and test prohibitive numbers of networking solutions. // relates to Petar's #2  
+ 
+**Some points for CNTT to consider when contemplating recommendations:**
+   > * Despite large variances in implementation, the spectrum of networking capabilities ultimately delivered to Workloads is comparatively narrow
+   > * Standard CNTT methodology (i.e. normalize interfaces, APIs, capabilities and behaviors at the reference points) applies well to networking, helping mitigate the need to be overly prescriptive about implementation  
+   > * CNTT/OPNFV MUST provide a functional networking solution for the RI, and in support of RC  
+   > * CNTT/OPNFV potentially does NOT have to provide a production networking solution for RI  
+   > * CNTT/OPNFV is NOT planning to provide a production compute solution for RI
+
+
+<a name="1.9.1"></a>
+## 1.9.1 Executive Summary
+
+> _Placeholder for the Executive Summary (targeting Baldy for first draft). **If you are interested in writing this content, please contact the RM lead.**_
+
+<a name="1.9.2"></a>
+## 1.9.2 Networking Strategy Objectives
+
+This section catalogs CNTT's high-level objectives for the Networking and Fabric Strategy. 
+
+> _List needs to be prioritized; expect additional objectives to be added, as they arise. This represents the _What_, not the _How_.
+
+1. The implementation of Networking inside the HW Layer should not be visible to the VNF/CNF and should preferably not even be visible to the IaaS/CaaS // TF #2
+1. Provide networks for L3 tenant, GWs, SDS, etc.
+1. CLEANLY decouple interface/reference points between CNTT constituencies
```

Comment:
Please remove all-caps use for "Cleanly".

Reply:
Done